benchmarks/ivpsolvers.py

```diff
@@ -20,12 +20,11 @@ class IVPSolve:
     params = [["eks0", "ekf0"], ["with", "without"], ["ibm4", "ioup4", "matern92"]]
 
     def setup(self, method, precond, prior):
-        # pylint: disable=attribute-defined-outside-init,invalid-name,unused-argument
+        # pylint: disable=invalid-name
```

Comment:
thanks. why are these not necessary anymore? Are they ignored globally?

Reply:
yes exactly.